### PR TITLE
Added Health and Safety Policy

### PR DESCRIPTION
**Pull request by yomiadetutu1**

### CODE REVIEW

**Review by natalie-hub — DISMISSED** [review]
LGTM

File: operations/workplace/people/people-policies.md [hunk 1]
```diff
@@ -49,3 +49,111 @@ Mattermost's Affirmative Action Plan can be found [here](https://docs.google.com
 Mattermost does not engage in electronic monitoring of staff or their activities through their work equipment, such as Mattermost-issued laptops or cell phones, work email, Mattermost, or other applications used by the company, except for legal, security, or compliance purposes. Any information collected will be used solely to help Mattermost meet its legal, security, or compliance requirements.
 
 In the event of a workplace investigation, Mattermost may request access to staff activities on  Mattermost-issued systems or devices, where necessary.
+
+### Health and Safety Policy
+
+Creating a safe workplace is a shared responsibility between Mattermost and all staff. Mattermost is committed to promoting a safe work environment and supporting everyone in ensuring their home work environment is safe, and adheres to legislative requirements for healthy working conditions.
+
+Mattermost’s  Health & Safety policy outlines the procedures and guidelines for conducting health and safety assessments for all staff. It  aims to:
```
Comment (cwarnermm) [inline operations/workplace/people/people-policies.md]:
```suggestion
Mattermost’s Health & Safety policy outlines the procedures and guidelines for conducting health and safety assessments for all staff. It aims to:
```

File: operations/workplace/people/people-policies.md [hunk 2]
```diff
@@ -49,3 +49,111 @@ Mattermost's Affirmative Action Plan can be found [here](https://docs.google.com
...
+* Prevent home office environment accidents and work-related ill health.
+* Manage health and safety risks associated with remote work.
+* Provide clear instructions, information and adequate training on workplace health and safety.
```
Comment (cwarnermm) [inline operations/workplace/people/people-policies.md]:
```suggestion
* Provide clear instructions, information, and adequate training on workplace health and safety.
```

File: operations/workplace/people/people-policies.md [hunk 3]
```diff
@@ -49,3 +49,111 @@ Mattermost's Affirmative Action Plan can be found [here](https://docs.google.com
...
+
+While we do not prescribe a specific way for staff to set up their work area, the expectation is that everyone maintains a comfortable and safe work environment in a manner that reduces hazards and enhances productivity. This includes the use of ergonomic items such as appropriate desks, chairs, proper lighting, clear walkways, organized electrical cables, and suitable equipment to minimize the risk of workplace injuries. Here’s a [guide](https://docs.google.com/document/d/13-48iC0fKJ17kEgqf798pp7lPGyHq_NzexgcKkSzFgI/edit?usp=sharing) on how to set up an ergonomic home office for maximum productivity and comfort. 
+
+**Initial Assessment**
+
+Prior to commencing work at Mattermost, FT staff will be asked to complete an initial health and safety self-assessment. The assessment will help staff to evaluate their home working environment, including workspace ergonomics, lighting, and any potential hazards.
+
+**Health and Safety Responsibilities**
+
+Building a safe workplace is a joint responsibility of all staff, managers and Mattermost. The rights and responsibilities of all parties are outlined below.
```
Comment (cwarnermm) [inline operations/workplace/people/people-policies.md]:
```suggestion
Building a safe workplace is a joint responsibility of all staff, managers, and Mattermost. The rights and responsibilities of all parties are outlined below.
```

File: operations/workplace/people/people-policies.md [hunk 4]
```diff
@@ -49,3 +49,111 @@ Mattermost's Affirmative Action Plan can be found [here](https://docs.google.com
...
+* Create and implement a health and safety policy.
+* Ensure that staff are aware of potential remote workplace health and safety hazards by providing information, instructions, and training where required.
+
+**Joint Health and Safety Committee (JHSC)**
+
+The JHSC's role is to raise awareness of health and safety issues in the workplace, identify health and safety risks, and make recommendations on areas for improving safety where necessary. The committee will hold regular meetings and initiate the quarterly workplace inspections conducted by staff members of their home office. Committee members include:
+
+| Name | Title |
+| :--- | :--- |
+| [Yomi Adetutu](https://community.mattermost.com/core/messages/yomi.adetutu) |People Generalist|
```
Comment (cwarnermm) [inline operations/workplace/people/people-policies.md]:
```suggestion
| [Yomi Adetutu](https://community.mattermost.com/core/messages/yomi.adetutu) | People Generalist |
```

**Review by cwarnermm — COMMENTED** [review]
@yomiadetutu1 - I've provided editorial feedback inline.

**Review by lynn915 — DISMISSED** [review]

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**